src/domain/octree/OctreePacketProcessor.ts

```diff
@@ -70,30 +72,50 @@ class OctreePacketProcessor {
      *  @function OctreePacketProcessor.addedEntity
      *  @returns {Signal}
      */
-    get addedEntity(): Signal {
-        return this.#_addedEntity.signal();
+    get entityData(): Signal {
+        return this.#_entityData.signal();
     }
 
     // Listener
-    // eslint-disable-next-line
-    // @ts-ignore
-    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {// eslint-disable-line
+    #processPacket = (message: ReceivedMessage, sendingNode: Node | null): void => {
         // C++ void OctreePacketProcessor::processPacket(QSharedPointer<ReceivedMessage> message, SharedNodePointer sendingNode)
 
-        const packetType = message.getType();
+        assert(sendingNode !== null, "OctreePacketProcessor.processPacket()", "Sending node is missing.");
 
-        if (packetType === PacketType.OctreeStats) {
-            console.error("OctreePacketProcessor: Packet type not processed: ", packetType);
-            // WEBRTC TODO: Address further C++ code.
-            return;
+        let messageLocal = message;
+
+        const octreePacketType = messageLocal.getType();
+
+        if (octreePacketType === PacketType.OctreeStats) {
+            // WEBRTC TODO: Address further C++ code - process OctreeStats packet.
+
+            console.error("OctreePacketProcessor: Packet type not processed: ", octreePacketType);
```

Comment:
```suggestion
```
Let's remove this console error. I don't think the OctreeStats packet is particularly important to address in the near future.